mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -148,21 +148,18 @@ func schemaNodeConfig() *schema.Schema {
 				"guest_accelerator": {
 					Type:     schema.TypeList,
 					Optional: true,
-					Computed: true,
```

Comment:
this will recreate when guest_accelerator is removed as there no method to remove via API (I tried thoroughly) but will allow updating if changes to nested args are made